pkg/util/ovs/ovs.go

```diff
@@ -15,12 +15,14 @@ import (
 type Interface interface {
 	// AddBridge creates the bridge associated with the interface, optionally setting
 	// properties on it (as with "ovs-vsctl set Bridge ..."). If the bridge already
-	// existed, it will be destroyed and recreated.
+	// exists this errors.
 	AddBridge(properties ...string) error
 
-	// DeleteBridge deletes the bridge associated with the interface. (It is an
-	// error if the bridge does not exist.)
-	DeleteBridge() error
+	// DeleteBridge deletes the bridge associated with the interface. The boolean
+	// that can be passed determines if a bridge not existing is an error. Passing
+	// true will delete bridge --if-exists, passing false will error if the bridge
+	// does not exist.
+	DeleteBridge(ifExists bool) error
```

Comment:
Do we ever call DeleteBridge with false?

Reply:
For now, we are not calling DeleteBridge with false anywhere.